totalRP3/core/impl/popup.lua

```diff
@@ -266,15 +266,15 @@ local TRP3_MusicBrowser = TRP3_MusicBrowser;
 local musicWidgetTab = {};
 local filteredMusicList;
 
-local function decorateMusic(lineFrame, musicURL)
-	musicURL = filteredMusicList[musicURL];
-	local musicName = musicURL:reverse();
-	musicName = (musicName:sub(1, musicName:find("%\\")-1)):reverse();
+local function decorateMusic(lineFrame, musicID)
+	local musicName = filteredMusicList[musicID][1];
+	local musicFile = filteredMusicList[musicID][2];
+	local musicDuration = filteredMusicList[musicID][3];
 
 	setTooltipForFrame(lineFrame, lineFrame, "RIGHT", 0, -30, musicName,
-	("|cff00ff00%s\n\n|cffff9900%s: |cffffffff%s\n|cffff9900%s: |cffffffff%s"):format(musicURL, loc.CM_L_CLICK, loc.REG_PLAYER_ABOUT_MUSIC_SELECT2, loc.CM_R_CLICK, loc.REG_PLAYER_ABOUT_MUSIC_LISTEN));
+	("|cff00ff00%s : %ss\n\n|cffff9900%s: |cffffffff%s\n|cffff9900%s: |cffffffff%s"):format(loc.UI_MUSIC_DURATION, floor(musicDuration + 0.5), loc.CM_L_CLICK, loc.REG_PLAYER_ABOUT_MUSIC_SELECT2, loc.CM_R_CLICK, loc.REG_PLAYER_ABOUT_MUSIC_LISTEN));
```

Comment:
The colon between the duration label and duration value has a space preceeding it, but none of the color label/value colons do.